Panic if PROJECT varaible not set

There is a [logic error in the main script](https://github.com/CrunchyData/crunchy-watch/blob/master/main.go#L212) that causes a panic if `KUBE_PROJECT` or `OSE_PROJECT` isn't set.

```
Starting crunchy watch
Started...
env var debug value is []
panic: runtime error: index out of range

goroutine 1 [running]:
main.main()
	/opt/cdev/src/github.com/crunchydata/crunchy-watch/main.go:218 +0x1237
```

Correct me if I'm wrong but `os.Args` always returns at least one value.  This should be `< 2`.

The `Dockerfile` run script should also enforce this variable being set.

--- a/main.go
+++ b/main.go
@@ -209,7 +209,7 @@ func main() {
 		}
 	}()
 
-	if len(os.Args) < 1 {
+	if len(os.Args) < 2 {
 		errorExit()
 	}
 

